packages/components/src/components/themed-scrollbars/themed-scrollbars.jsx

```diff
@@ -1,28 +1,64 @@
+import classNames from 'classnames';
 import React from 'react';
-import { Scrollbars } from 'tt-react-custom-scrollbars';
-
-class ThemedScrollbars extends React.Component {
-    render() {
-        if (this.props.is_native) return this.props.children;
-        return (
-            <Scrollbars
-                ref={this.props.list_ref}
-                renderTrackHorizontal={props => (
-                    <div
-                        {...props}
-                        className='dc-themed-scrollbars__track--horizontal'
-                        style={{ display: this.props.hideHorizontal ? 'none' : 'block' }}
-                    />
-                )}
-                renderTrackVertical={props => <div {...props} className='dc-themed-scrollbars__track--vertical' />}
-                renderThumbHorizontal={props => <div {...props} className='dc-themed-scrollbars__thumb--horizontal' />}
-                renderThumbVertical={props => <div {...props} className='dc-themed-scrollbars__thumb--vertical' />}
-                {...this.props}
-            >
-                {this.props.children}
-            </Scrollbars>
-        );
-    }
-}
+
+const ThemedScrollbars = ({
+    children,
+    className,
+    height,
+    width,
+    autohide = true,
+    is_native,
+    is_only_horizontal,
+    has_horizontal,
+    onScroll,
+    refSetter,
+}) => {
+    if (is_native) return children;
+    const useHover = () => {
+        const [value, setValue] = React.useState(false);
+
+        const ref = refSetter || React.useRef(null);
+
+        const handleMouseOver = () => setValue(true);
+        const handleMouseOut = () => setValue(false);
+
+        React.useEffect(() => {
+            const node = ref.current;
+            if (node) {
+                node.addEventListener('mouseover', handleMouseOver);
+                node.addEventListener('mouseout', handleMouseOut);
+
+                return () => {
+                    node.removeEventListener('mouseover', handleMouseOver);
+                    node.removeEventListener('mouseout', handleMouseOut);
+                };
+            }
+            return null;
+        }, [ref.current]);
+
+        return [ref, value];
+    };
```

Comment:
Can we move this out of the component? Looks like a good reusable effect.

Reply:
@reza-fs , for now I moved the hook effect out as a helper file for `themed-scrollbars` as currently we don't have anywhere in components where we're storing helpers or reusable functions. 

I considered shared but we can't move it there as we want to avoid adding `react` as a dependency in that package.

Maybe we can refactor the components and add a shared helper file within the components package for all possible reusable `effects`.

Let's do this in a separate PR as we'll need to refactor other components as well.